02/app.js

```diff
@@ -1,2 +1,27 @@
 const x = 5;
-let result = 0;
\ No newline at end of file
+let result = 0;
+
+// for loop
+console.log('pętla FOR')
+
+for (let i = 1; i <= x; i++) {
+    result = result + i;
+    console.log(result)
+}
+
+console.log('Wynik końcowy pętli for: ' + result)
+
+
+// while loop
+console.log('pętla WHILE')
+
+let j = 0;
+result = 0;
+
+while (j < x) {
+    j++;
```

Comment:
Zwróć uwagę, że tutaj warunek i kolejność operacji ma znaczenie.
Zadanie poprawnie wykonane - chciałem, abyś tylko zwrócił na to uwagę.

Reply:
ok